server/devdocs/bin/generate-proptypes-index.js

```diff
@@ -115,14 +209,15 @@ const main = ( () => {
 		} );
 
 	if ( fileList.length === 0 ) {
-		process.stderr.write( 'You must pass a list of files to process' );
-		process.exit( 1 );
+		throw new Error( 'You must pass a list of files to process' );
 	}
 
-	const documents = createIndex(
-		fileList
-			.map( processFile )
-			.map( parseDocument )
-	);
+	let documents = fileList.map( processFile );
+	documents = onlyWithExample( documents );
+	documents = documents.map( parseDocument );
+	documents = cleanIndex( documents );
```

Comment:
I swear I learn something new from lodash once a month. That's pretty excellent and simple.